deploy/manifests/prod/us-east-2/tenant/storetheindex/assigner/config.json

```diff
@@ -8,34 +8,27 @@
     "FilterIPs": false,
     "IndexerPool": [
       {
-        "AdminURL": "http://dido-indexer:3002",
```

Comment:
Are we not going be assigning to the existing indexers anymore?

Reply:
No. We want any new publishers to get assigned to the new indexers, and not to the old ones.

Reply:
Got it, thank you